src/firebase/firestore/index.ts

```diff
@@ -42,43 +43,42 @@ export const getUserRoleData = async (uid: string | undefined): Promise<string |
 	}
 }
 
-/**
- * Function creates a new document in "users" collection where sets all new user's data;
- *
- * @param {ROLES} role - new user's role in the system (admin or garson);
- * @param {string} uid - new user's uid;
- */
-export const createNewUser = async (userData: NewUserProps): Promise<void> => {
-	try {
-		await addDoc(collection(db, 'users'), { role: userData.role, name: userData.name, uid: auth.currentUser?.uid })
-	} catch (error) {
-		throw new Error(`AN ERROR OCCURED: ${error}`)
-	}
-}
-
-/**
- * Function creates a new user via EMAIL and PASSWORD;
- *
- * @param {userData} role - object with new user's data;
- */
+// export const getUserRoleData = async (uid: string | undefined): Promise<string | undefined> => {
+// 	let currentUserRole
+// 	const docRef = doc(collection(db, 'users'), auth.currentUser?.uid)
+// 	const docSnap = await getDoc(docRef)
+// 	const docData = docSnap.data();
+// 	console.log(auth.currentUser?.uid);
+// 	console.log(docData);
+// 	currentUserRole = docData?.role
+// 	return currentUserRole
+// }
 
 export const createNewUserWittEmailAndPassword = async (userData: NewUserProps): Promise<void> => {
 	createUserWithEmailAndPassword(auth, userData.email as string, userData.password as string)
```

Comment:
no, that doesn't look right. looks like you're using duplicated function. leave only one of them

Reply:
Comment there was deleted, but last commit didn`t includes this file by some magic